### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by nick-zh**

### CODE REVIEW

File: doc/client.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ There are 3 clients: `FileGetContents`, `Curl` and `MultiCurl`.
 ```php
 $request = new PSR7Request('GET', 'https://example.com');
 
-$client = new Buzz\Client\FileGetContents(new Psr17ResponseFactory(), ['allow_redirects' => true]);
+$client = new Buzz\Client\FileGetContents(['allow_redirects' => true], new Psr17Factory());
```
Comment (Nyholm) [inline doc/client.md]: This has changed in dev-master. It was correct before your fix.

**Comment by Nyholm** [comment]
Thank you. For fixing this.

File: doc/client.md (hunk 1 above)
Comment (nick-zh) [reply doc/client.md]: Ah ok perfect, sry must have missed it :)

**Comment by Nyholm** [comment]
I released 1.0.0 not. Try it out :)

**Pull request merged**